src/services/storage/drivers/redis/types.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface RedisStorageDriverConfig {}
```

Comment:
The Redis driver doesn't need any configuration? what about the URL? possible credentials?